.github/workflows/_test_upstream_pax.yaml

```diff
@@ -210,6 +221,14 @@ jobs:
             BATCH_SIZE: 4
             EVALUATE: true
             ADDITIONAL_ARGS: "--model-type LLaMA70BProxy --evaluate"
+          - TEST_NAME: 2DP1FSDP1TP4PP_fmha
+            PARALLEL_CONFIG: [4, 2, 1, 1]
```

Comment:
Is this redundant now that fmha is enabled by default?

Reply:
Yeah, I'll use the existing test configs (1-2 tests ) to add additional arg to save hlo (used to debug fmha instructions). Don't want to enable hlo dumping by default. (Terry mentioned it takes a lot of time in nightly to download artifacts. )